ci-operator/step-registry/openshift/upgrade/qe/test/disconnected/cert-rotation/openshift-upgrade-qe-test-disconnected-cert-rotation-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
```

Comment:
Looks like this ref is not used, do we still want to keep it?

Reply:
It should be deleted.